package/ubuntu/hypercontainer/debian/changelog

```diff
@@ -0,0 +1,5 @@
+hypercontainer (0.6.2-1) Xenial Xerus; urgency=low
```

Comment:
IIRC, the `distribution` field should be a single word, like this ([soruce](http://packaging.ubuntu.com/html/debian-dir-overview.html#the-changelog))

```
hello (2.8-0ubuntu1) trusty; urgency=low

  * New upstream release with lots of bug fixes and feature improvements.

-- Jane Doe <packager@example.com>  Thu, 21 Oct 2013 11:12:00 -0400

```